08LessonEight/homework/css/style.css

```diff
@@ -0,0 +1,47 @@
+/*Add CSS rules*/
+body{
+  display: flex;
+  width:100%;
+  justify-content: center;
+  font-family: sans-serif;
+}
+main{
+  width:960px;
+  display: flex;
+  flex-flow:row wrap;
+}
+.col{
+  width:31%;
+  box-sizing: border-box;
+  margin:1%;
+}
+.click{
+  padding:10px 15px;
+  border: 1px solid gray;
+  border-right:4px solid red;
+  font-weight: 700;
+  font-size: 12px;
+}
+.click:hover{
+  border:white;
+  background-color:red;
+  color:white;
+}
+.col h1{
+  margin-top: 30px;
+  font-size: 14px;
+  font-weight: 700;
+  color:#36454f;
+}
+.col p{
+  font-size: 10px;
+  color: gray;
+  margin: 12% 0;
+  line-height: 18px;
+}
+.col span{
+  color:gray;
+}
+main img{
+  width:100%;
+}
```

Comment:
CSS looks great! Way to start with the most generic at top and get more specific going down. Also good to group similar "sections" as you did (keeping all the `.col` styles together).